packages/clean-tag/package.json

```diff
@@ -30,5 +29,8 @@
       "babel-register"
     ],
     "babel": "inherit"
+  },
+  "peerDependencies": {
+    "react": "^0.14.x || ^15.x || ^16.x"
```

Comment:
I think `React.forwardRef` requires 16.x (can't remember which minor version): https://github.com/styled-system/styled-system/blob/master/packages/clean-tag/index.js#L20

Reply:
Thanks for looking into this!

https://github.com/facebook/react/blob/master/CHANGELOG.md#1630-march-29-2018

16.3 for `forwardRef `